draft-ietf-trans-rfc6962-bis.md

```diff
@@ -637,16 +637,22 @@ following requirements:
   that the Transparency Information ({{x509v3_transinfo_extension}}) extension
   MUST be omitted.
 
-* `SignedData.signerInfos` MUST contain a signature from the same (root or
-  intermediate) CA that will ultimately issue the certificate. This signature
-  indicates the CA's intent to issue the certificate. This intent is considered
-  binding (i.e., misissuance of the precertificate is considered equivalent to
-  misissuance of the certificate). (Note that, because of the structure of CMS,
-  the signature on the CMS object will not be a valid X.509v3 signature and so
-  cannot be used to construct a certificate from the precertificate).
+* `SignedData.signerInfos` MUST contain 1 `SignerInfo` with a `signature` from
```

Comment:
I like the idea of limiting this further, because that would make log implementation a bit simpler.

However, that would mean overriding this requirement in https://tools.ietf.org/html/rfc5652#section-5.3...
`      Implementations MUST support the reception of the
      issuerAndSerialNumber and subjectKeyIdentifier forms of
      SignerIdentifier.`
Can we do that?